Library/Formula/agda.rb

```diff
@@ -51,38 +58,61 @@ def install
     cabal_clean_lib
 
     if build.with? "stdlib"
-      resource("stdlib").stage prefix/"agda-stdlib"
+      resource("stdlib").stage lib
```

Comment:
`skip_clean` stops files being deleted but does not stop them being linked. The only way to do that is to put them in a different path than `lib`.